IR_Turret/IR_Turret.ino

```diff
@@ -29,144 +30,162 @@ unsigned long previousMillis = 0;
 
 String mode = "IDLE";
 
-
 void setup(void) {
-  Serial.begin(9600);
+  Serial.begin(115200);
+  while (!Serial)
+    ; // Wait for Serial to become available.
   Serial.println("Power On.");
   Serial.println(" ");
   tiltservo.attach(tiltServoPin);
+  delay(100);
   tiltservo.write(tiltPosition);
-  delay(50);
+  delay(100);
+  tiltservo.detach();
```

Comment:
I'm not sure why the original code maintained control to the tiltservo. It seems unnecessary and in my case was causing enough noise on the servo board that it was registering random IR codes and preventing me from proper IR control.